apps/openmw/mwmechanics/pathfinding.cpp

```diff
@@ -344,7 +346,7 @@ namespace MWMechanics
             const auto world = MWBase::Environment::get().getWorld();
             const auto stepSize = getPathStepSize(actor);
             const auto navigator = world->getNavigator();
-            navigator->findPath(halfExtents, stepSize, startPoint, endPoint, flags, out);
+            return navigator->findPath(halfExtents, stepSize, startPoint, endPoint, flags, out).is_initialized();
         }
         catch (const DetourNavigator::NavigatorException& exception)
         {
```

Comment:
This path does not not return value, despite the function returns boolean value.

Reply:
Thanks, missed this.

Reply:
Are you sure that it should return "true" instead of "false"? 
As I understood, if an attempt to build path leads to exception, the resulting path may be an incorrect, and in this case  it makes sense to fallback to straight path as well.

Reply:
No, it should be true, because it indicates the fact that there was a try to find a path by navmesh. If the try is failed then the destination can’t be used for wandering actor. If it will be false then actor will wander to any destination and use of navmesh in AiWander will became useless.